client/my-sites/plugins/main.jsx

```diff
@@ -127,7 +128,11 @@ export default React.createClass( {
 	addWporgDataToPlugins( plugins ) {
```

Comment:
yup. Or even a selector, once everything is redux-ed. But this method is merging data we have in our sites-list component, which is not even a flux store, so until that one is migrated to something which supports listening to actions, there's not a lot we can improve here...